### ISSUE TITLE
Update content for supply re-order sign-in alert widget 

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#94862]
## Description

Some slight changes have been made to the global versions of sign-in alerts that will eventually be included as part of VADS. In the future, we should be able to just take the VADS component, but the MHV sign in CTA widget needs a content update in the meantime to meet our mid-Dec target for having VADS-compliant sign-in widgets on unauth pages for all health tools, including the supply reorder page. 

### User story

As a Veteran, I want to see the exact same content in the sign-in widget for supply reordering as I see when I encounter sign-in alerts anywhere else on VA.gov, so that I trust VA.gov as a consistent, reliable site that is communicating the same information to me no matter where I go. 

### Notes
- VADS sign-in widget components (finalized in Figma) [here](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=13356-315&node-type=canvas&t=rIiRYmTBppXUaxIK-0) - **still undergoing coding** and aren't usable by us yet.
- We will still need the slightly modified alert content for the existing widget
- Slack thread regarding updates [here](https://dsva.slack.com/archives/C0581MN69TJ/p1728495597447689) 

### Content update: 
~**Heading:** No change to the code, but change the README to show the updated sample service description of `re-order medical supplies` in the template.~

~**Body:**~

_[REFER to VADS PR/documentation]_

### Possible tasks:
- [ ] Update the README with the updates service description
- [ ] make minor changes (written out above) to the alert body to match the global sign-in version that was recently finalized. (images below for posterity if helpful)

**Current design** (screengrabbed from [PR here](https://github.com/department-of-veterans-affairs/vets-website/pull/31255)) 
<img width="1204" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOWsvQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9f190956ddf65353941c5280129afb350b18afff/Screenshot%202024-10-11%20at%2011.13.13%E2%80%AFAM.png" alt="Screenshot 2024-10-11 at 11.13.13 AM.png" />

**What it should be after content modifications:** NOTE: This image is shown without the service description in the heading, but the service description still applies.
<img width="1594" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXMvQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1a9fb13be05bb0982765e6559052b9709f8c9562/Screenshot%202024-10-11%20at%2011.15.45%E2%80%AFAM.png" alt="Screenshot 2024-10-11 at 11.15.45 AM.png" />

## Acceptance criteria
- [ ] ~MHV Sign-in CTA README shows the updated sample service description.~
- [ ] Sign-in widget on the unauthed supply reorder page has content for signing in and verifying accounts that is exactly like the VADS version that we implemented in the other widgets:
  - [ ] Sign-in content
  - [ ] "Verify" for login.gov
  - [ ] "Verify" for ID.me
  - [ ] "Use a different credential" for MHV Basic
- [ ] A user signed in with a verified, modern credential sees the link to the application
- [ ] ¿¿Except for the heading, which is still custom to specify the purpose of ordering supplies
- [x] QA review as appropriate (Staging or a Drupal preview env)

**Comment by LynneJohnson224** [comment]
@allanto-ah is QA on this blocked by the same thing blocking the other sign in widget tickets?

**Comment by allanto-ah** [comment]
@LynneJohnson224 yes, will move this to Blocked, this and [89275](https://github.com/department-of-veterans-affairs/va.gov-team/issues/89275) will be QA'ed at the same time

**Comment by wesrowe** [comment]
@allanto-ah @nichia, anyone know if we can get this unblocked? The comments above say "same thing" and "same time", but I don't remember the "as what".

**Comment by sterkenburgsara** [comment]
The supply re-order alert has a custom heading in it, which is why it was held up. (I think?) But now we have to scale this unauth page to support the form expanding to way more supplies, and I don't think we do need the custom version anymore. I need @laurwill and @Jenny-yw to check my memory, though.

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the sign in widgets have been put into the proper app pages. Currently there is a "i" icon instead of the lock icon. The "i" icon will be replaced with a lock icon once the actual VADS component is implemented

Example screenshot
<img width="1309" alt="Image" src="https://github.com/user-attachments/assets/c0a7eff6-3e2f-44d6-a052-eac085a868f7" />

**Comment by Jenny-yw** [comment]
@allanto-ah Was this ticket for supplies closed accidentally? I don't see the new sign in alert in staging on the supplies page. https://staging.va.gov/health-care/order-medical-supplies/

**Comment by allanto-ah** [comment]
@Jenny-yw ah yeah, let me re-open, sorry about that

**Comment by wesrowe** [comment]
Additional info: this one has to be checked in a "Drupal preview environment." I don't think we can test this until CAIA team is back on the job.

**Comment by wesrowe** [comment]
And here is the [CAIA ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499) involved with this request. I think the history of this widget is spread across multiple tickets. Hopefully these past two comments pull thing together into the right place.

**Comment by wesrowe** [comment]
Here's [a key comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499#issuecomment-2441601595) from Laura on the CAIA ticket:

>Hey Wes, if you have the widget code ready, I can add it to the page — we usually test widget content in Drupal preview in prod, but I can try adding it to staging and see if that works.

It looks like this PR is merged, so CAIA can test it wherever they want (prod).

**Comment by wesrowe** [comment]
Pinged Laura [in Slack](https://dsva.slack.com/archives/C0581MN69TJ/p1736801646969559) since she didn't respond to my comment on the ticket (above).

**Comment by wesrowe** [comment]
Laura found that the header level is wrong for the page ([slack thread](https://dsva.slack.com/archives/C0581MN69TJ/p1736871977116899?thread_ts=1736801646.969559&cid=C0581MN69TJ)). It's currently an H2.

Alex Garcia confirmed heading level is configurable.

It needs to be an H4, as the "order online" directly above it is an H3: 

<img width="664" alt="Image" src="https://github.com/user-attachments/assets/e8c7c347-9415-4100-b238-e644160e289a" />

**Comment by wesrowe** [comment]
@mcarterA6, given that the CAIA contract changed hands as of Monday, I'm thinking we need a new ticket to make sure this widget gets published once we're done fixing the heading.

I'm going to write one in the March 15 widget epic.

**Comment by wesrowe** [comment]
@nichia, have you doubled back to update the header levels for this widget as commented above?

**Comment by nichia** [comment]
Sorry, not yet. Still need to add a user customizable header level to the widget

**Comment by wesrowe** [comment]
I took this header-level question [to slack](https://dsva.slack.com/archives/C0581MN69TJ/p1738267642865969).

**Comment by nichia** [comment]
Modify _mhv-signin-cta_ to add a '_data-heading-level_' attribute that static-content people can set the heading level for headings within the widget. 
Set the default value for heading level to 2 - pages that requires a different widget heading level then the default can modify it via this example snippet _data-heading-level="4"_

**Comment by wesrowe** [comment]
@nichia, I happened to see that your PR merged. Are we ready to ask CAIA (if they're available) to test this in a Drupal preview env?

**Comment by nichia** [comment]
Thanks @wesrowe, yes, the PR is merged and ready for CAIA to test in a Drupal preview env.
The alert heading level default is `h3` - to use 'h4' alert heading level, CAIA will need to add this attribute `data-heading-level="4"`

**Comment by wesrowe** [comment]
We have a snag. Will attend CAIA office hours today.
